.golangci.yml

```diff
@@ -17,6 +17,7 @@ run:
   # default value is empty list, but default dirs are skipped independently
   # from this option's value (see skip-dirs-use-default).
   skip-dirs:
+  - third_party
```

Comment:
Note this does a regex match and will skip any directory whose path includes "third_party".

Alternatively I could make this more explicit by putting something like:
```yaml
- ^receiver[/|\\]hostmetricsreceiver[/|\\]internal[/|\\]third_party
```
(unfortunately `[/|\\]` is needed to support different OS path separators)